aicsimageio/readers/sldy_reader/__init__.py

```diff
@@ -0,0 +1,301 @@
+        Default: {}
+    data_file_prefix: str, default = "ImageData"
+        Prefix to the data files within the image directories to extract.
+        By default this will
+        be set to "ImageData". However, specifying "HistogramData" would allow you
+        to interact with the image's histogram data instead.
+
+    Raises
+    ------
+    exceptions.UnsupportedFileFormatError
+        If the file is not supported by this reader.
+    """
+
+    @staticmethod
+    def _is_supported_image(fs: AbstractFileSystem, path: str, **kwargs: Any) -> bool:
+        try:
+            SldyReader._get_images_from_data_directory(
+                fs, path, kwargs.get("data_file_prefix", DEFAULT_DATA_FILE_PREFIX)
+            )
+            return True
+        except Exception:
+            return False
+
+    @staticmethod
+    def _get_images_from_data_directory(
+        fs: AbstractFileSystem, path: types.PathLike, data_file_prefix: str
+    ) -> List[SldyImage]:
+        """
+        Retrieves list of image acquisitions found at path.
+
+        Parameters
+        ----------
+        fs: AbstractFileSystem
+            The file system to used for reading.
+        path: str
+            The path to the file to read.
+        data_file_prefix: str
+            Prefix to the data files within the image directories to extract.
+
+        Returns
+        -------
+        images: List[SldyImage]
+            A list of the acquisitions found at the path given
+            ordered by their names
+        """
+        data_directory = Path(path).with_suffix(".dir")
+        images = [
+            SldyImage(fs, image_dir, data_file_prefix=data_file_prefix)
+            for image_dir in data_directory.glob("*.imgdir")
+        ]
+        if not images:
+            raise ValueError("Unable to find any images within the image directory")
+
+        # Prevent inconsistent scene (image) ordering
+        images.sort(key=lambda img: img.id)
+        return images
+
+    def __init__(
+        self,
+        image: types.PathLike,
+        fs_kwargs: Dict[str, Any] = {},
+        data_file_prefix: str = DEFAULT_DATA_FILE_PREFIX,
+        **kwargs: Any,
+    ):
+        # Expand details of provided image
+        self._fs, self._path = io_utils.pathlike_to_fs(
+            image,
+            enforce_exists=True,
+            fs_kwargs=fs_kwargs,
+        )
+
+        try:
+            self._images = SldyReader._get_images_from_data_directory(
+                self._fs, self._path, data_file_prefix=data_file_prefix
+            )
+        except Exception:
+            # Enforce valid image
+            raise exceptions.UnsupportedFileFormatError(
+                self.__class__.__name__, self._path
+            )
+
+    @property
+    def scenes(self) -> Tuple[str, ...]:
+        if self._scenes is None:
+            self._scenes = tuple(
+                metadata_utils.generate_ome_image_id(image.id) for image in self._images
```

Comment:
do they not store scene names in the metadata somewhere?

Reply:
I did not see it anywhere so I used the names as you suggested [`id` refers to [this](https://github.com/AllenCellModeling/aicsimageio/blob/f0af454cbead1a78468508b00abc695369772f13/aicsimageio/readers/sldy_reader/sldy_image.py#L177) which is a bit ugly but 🤷 )

Reply:
i see, yeah.

Reply:
I think the thing is, the generate_ome... functions are applying a OME-specific convention which doesn't need to apply to sldy.  The scene name is just the name and doesn't need the `Image:` prefix.  Same for `Channel:`.

Reply:
Ah okay I thought the idea was to try to standardize arbitrary metadata into OME. I'll remove the prefixes.

Reply:
Done in 4f9b4c249edb6d32745bf0054f8c522ab5d351ba